database/schema.sql

```diff
@@ -35,6 +27,7 @@ CREATE TABLE region (
 CREATE TABLE constellation (
     constellation_id SMALLINT GENERATED ALWAYS AS IDENTITY,
     constellation_name VARCHAR(35) NOT NULL,
```

Comment:
Made add unique constraints to all name variables in the ERD?
```suggestion
    constellation_name VARCHAR(35) NOT NULL UNIQUE,
```